src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/submissionApiDelegateImpl/GenerateUrlTest.java

```diff
@@ -88,9 +90,8 @@
     @Mock
     private GenerateUrlRequestValidator generateUrlRequestValidatorMock;
 
-
-    @BeforeAll
-    public void beforeAll() {
+    @BeforeEach
```

Comment:
Changed @BeforeAll to @BeforeEach so that each unit test defined in Test Suite could be run independently. This change and other changes added were to ensure that if a test was renamed or if a test was to be run independently, then the test would not fail because a setup was changed.  This change would occur if test was dependent on values set when another test was run.